packages/odyssey-react/src/components/Form/Form.module.scss

```diff
@@ -11,22 +11,22 @@
  */
 
 .root {
-  max-width: calc(#{$max-line-length} + (2 * #{$spacing-m}));
-  margin-block-start: 0;
-  margin-block-end: $spacing-m;
-  margin-inline: 0;
-  padding-block: $spacing-m;
-  padding-inline: $spacing-m;
+  max-width: calc(var(--MaxWidthBase) + (2 * var(--MaxWidthBuffer)));
+  margin-block-start: var(--SpaceMarginBlockStart);
+  margin-block-end: var(--SpaceMarginBlockEnd);
+  margin-inline: var(--SpaceMarginInline);
+  padding-block: var(--SpacePaddingBlock);
+  padding-inline: var(--SpacePaddingInline);
 
   &:last-child {
-    margin-block-end: 0;
+    margin-block-end: var(--SpaceMarginBlockEndLastChild);
```

Comment:
It might be safe to leave this as a static `0` - not sure if we'd want someone to alter the behavior here.